Tools/autotest/sim_vehicle.py

```diff
@@ -745,6 +755,7 @@ def start_mavproxy(opts, stuff):
 group_sim.add_option("-m", "--mavproxy-args", default=None, type='string', help="additional arguments to pass to mavproxy.py")
 group_sim.add_option("", "--strace", action='store_true', default=False, help="strace the ArduPilot binary")
 group_sim.add_option("", "--model", type='string', default=None, help="Override simulation model to use")
+group_sim.add_option("", "--use-dir", type='string', default=None, help="Store and launch SITL in named the directory")
```

Comment:
Description could do with a little bit of help.  Pretty sure we don't "Store SITL" in there....